<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-Q496GAK" ignored="0" name="Test2" tests="5" failures="2" timestamp="17 Feb 2021 11:40:54 GMT" time="236.091" errors="0">
  <testcase name="Customer_Type_Residential" time="46.964" classname="Marketing_Project.Market_All_Cusotmer"/>
  <testcase name="Customer_Type_Business" time="31.165" classname="Marketing_Project.Market_All_Cusotmer"/>
  <testcase name="Customer_Type_Commercial" time="41.923" classname="Marketing_Project.Market_All_Cusotmer"/>
  <testcase name="Add_Provisional_Service" time="42.208" classname="Marketing_Project.Market_All_Cusotmer">
    <failure type="java.lang.AssertionError" message="expected [NMI4365121] but found [MA81470942]">
      <![CDATA[java.lang.AssertionError: expected [NMI4365121] but found [MA81470942]
at Marketing_Project.Market_All_Cusotmer.Add_Provisional_Service(Market_All_Cusotmer.java:387)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Add_Provisional_Service -->
  <testcase name="customer_Save_nd_Add_Provisioning_option" time="58.476" classname="Marketing_Project.Market_All_Cusotmer">
    <failure type="java.lang.AssertionError" message="expected [NMI4365435] but found [NMI4365643]">
      <![CDATA[java.lang.AssertionError: expected [NMI4365435] but found [NMI4365643]
at Marketing_Project.Market_All_Cusotmer.customer_Save_nd_Add_Provisioning_option(Market_All_Cusotmer.java:537)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- customer_Save_nd_Add_Provisioning_option -->
</testsuite> <!-- Test2 -->
